plugins/huggingface/api_token.go

```diff
@@ -0,0 +1,88 @@
+package huggingface
+
+import (
+	"context"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func APIToken() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.APIToken,
+		DocsURL:       sdk.URL("https://huggingface.co/docs/huggingface_hub/quick-start"), 
```

Comment:
Let's update the documentation link to https://huggingface.co/docs/hub/security-tokens because this part is describing the token itself and not the CLI.

Reply:
Updated this, thanks